data/os/RedHat/9.yaml

```diff
@@ -0,0 +1,2 @@
+---
+powerdns::pgsql_schema_file: /usr/share/doc/pdns-backend-postgresql/schema.pgsql.sql
```

Comment:
As far as I can see, all RedHat-based packages have `/usr/share/doc/pdns-backend-<name>/schema.<backend>.sql` available and we can just update the paths in `data/os/RedHat.yaml`

Reply:
Makes sense to me.